.git-blame-ignore-revs

```diff
@@ -6,3 +6,9 @@ f63053cace3c02e284f00918e1854284c85b9132
 
 # Prettier upgrade to 2.6.2.
 33d84b036592a5bf31af05b7710f3b2b14163dc4
+
+# Prettier upgrade to 2.8.5.
+c56e8a1910ed74f405b74bbb12fe81dea974e5c3
```

Comment:
Thanks for adding it retroactively 👍

Reply:
That's actually the only way to do it, because we need to have the commit sha, which isn't available until we land it.